NAMESPACE

```diff
@@ -108,6 +133,9 @@ export(ml_random_forest)
 export(ml_save)
 export(ml_survival_regression)
 export(na.replace)
+export(print_jobj)
+export(register_extension)
```

Comment:
Similarly, should this become `sparklyr_register_extension`?

It's a shame that there isn't a nice way to enforce certain functions must be called through `::` as `sparklyr::register_extension` is nice as an interface, but it's not necessarily a function we want to make available after attaching `sparklyr`.